temporian/core/operators/sum.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sum operator."""
+
+from typing import Optional
+
+from temporian.core import operator_lib
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class SumOperator(Operator):
+    """Sum operator."""
+
+    def __init__(
+        self,
+        data_1: Event,
+        data_2: Event,
```

Comment:
Yes. I used those names because they were the ones chosen in the notes for the operator implementation. I agree that `event_1` and `event_2` are better names!